### ISSUE TITLE
in some CLIs explain :: syntax for expected values column

### ISSUE THREAD

**Issue by sergpolly** [open2c/cooltools#333]
@gfudenberg :
for CLI, maybe mention that the expected column specification is optional & provide an example? E.g. if the following is correct: ```
The path to a tsv-like file with a compatible expected table. Use the '::' syntax to optionally specify a column name (e.g. '/path/to/expected::balance.avg').```

this is applicable to `saddle`, `dots`, something else (`pileups`)?

**Comment by gfudenberg** [comment]
yes... probably should add it other places where relevant. maybe best as an
issue, then?

**Comment by sergpolly** [comment]
yep yep it's a whole separate issue now - so that we'll remember
